tests/CMakeLists.txt

```diff
@@ -0,0 +1,120 @@
+cmake_minimum_required(VERSION 3.13)
+# https://github.com/google/oss-policies-info/blob/main/foundational-cxx-support-matrix.md#foundational-c-support
+
+project(sunshine_tests)
```

Comment:
Should we possibly change the target names to just `tests` instead of `sunshine_tests`?

Reply:
renamed to `test_sunshine`